adzerk/transform.py

```diff
@@ -116,7 +116,8 @@ def get_personalization_models(body):
     else:
         # Topics in AdZerk prefixed with topic_ correspond with models in Firefox prefixed with nb_model_.
```

Comment:
Let's fix this comment too

Reply:
I think this comment is still accurate, but there is logic in Firefox the prepends the `nb_model_` prefix, is my understanding.
https://searchfox.org/mozilla-central/source/browser/app/profile/firefox.js#1374

Reply:
The files downloaded from the sever are nb_model_foo.

Internally to those files are the tags. The tags are simply too. It is these tags that are used inside the logic.